backend/dataall/core/permissions/db/resource_policy/resource_policy_repositories.py

```diff
@@ -3,10 +3,11 @@
 
 from sqlalchemy.sql import and_
 
-from dataall.core.permissions.db.permission_repositories import Permission
-from dataall.core.permissions.db.permission_models import PermissionType
+from dataall.core.permissions.db.permission.permission_repositories import Permission
+from dataall.core.permissions.api.enums import PermissionType
 from dataall.base.db import exceptions
-from dataall.core.permissions.db import permission_models as models
+from dataall.core.permissions.db.permission import permission_models
+from dataall.core.permissions.db.resource_policy import resource_policy_models as models
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Looking at this db repository class, it contains a lot of business logic. Maybe we can add a `resource_policy_service.py` service and keep the db package limited to db operations.

Reply:
For example, methods such as: `check_user_resource_permission`, `update_resource_policy`, `attach_resource_policy`... are not db related, they are purely business logic